docs/source/metrics.rst

```diff
@@ -63,14 +64,64 @@ It puts a greater influence on large errors than small errors, which makes this
 .. math::
     RMSE = \sqrt{\overline{(f - o)^{2}}}
 
+.. autofunction:: _rmse
+
+Mean Absolute Error (MAE)
+-------------------------
+
+``keyword: 'mae'``
+
+``perfect score: 0``
+
+The average of the absolute differences between forecasts and observations [2]_. A more robust measure of forecast accuracy than root mean square error or mean square error which is sensitive to large outlier forecast errors [3]_.
+
+.. math::
+    MAE = (\overline{\vert f - o \vert})
+.. autofunction:: _mae
+
+
+Derived Metrics
+===============
+
+Normalization based on comparison type. Comparison versus mean requires different normalization than comparison against every member (see ``climpred.metrics._get_norm_factor``).
+
+.. autofunction:: _ppp
+.. autofunction:: _nmse
+.. autofunction:: _nmae
+.. autofunction:: _nrmse
+
+
+Murphy decomposition metrics
+============================
+
+:cite:`Murphy1988`
+
+.. autofunction:: _std_ratio
+.. autofunction:: _bias
+.. autofunction:: _bias_slope
+.. autofunction:: _conditional_bias
+.. autofunction:: _msss_murphy
+
+
+Other metrics
+=============
+
+.. autofunction:: _less
+
+
 Probabilistic
 #############
 
+.. autofunction:: _crps
+.. autofunction:: _crpss
+
+
+.. bibliography:: refs.bib
+
+below old: to be transfered to refs.bib or deleted
```

Comment:
Don't forget to handle / delete this comment

Reply:
I will probably delete the refs.bib approach.